pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -497,7 +497,7 @@ def make_package(args):
     # Find the SDK directory and target API
     with open('project.properties', 'r') as fileh:
         target = fileh.read().strip()
-    android_api = target.split('-')[1]
+    android_api = int(target.split('-')[1])
     try:
         int(android_api)
```

Comment:
```suggestion
    android_api = target.split('-')[1]
    try:
        android_api  = int(android_api)
```

Reply:
Just after this call, we have a `try/except` block that checks if the `android_api` string value is parseable as an int.

Parsing the int from string before that call will lead to a crash, without raising the needed error.